### PR TITLE
Task/fix imports

### PR DESCRIPTION
**Pull request by auraz**
Originally:

1 ) Add python2 related imports 
2) Add requirements.txt which is installed when package installed

Jira ticket: https://liv-it.atlassian.net/browse/WEB-3789

But it looks, that we could have either 1) mock in prod and requirements for python2 2) test_compat and no mock in prod as a dependency. We decided to go 2).

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,21 +9,21 @@
...
-    license='BSD License',  # TODO which license to use
-    description='OAuth2 client for service to service communications',
+    license='MIT License',
+    description='OAuth2 Client for service to service communications',
     long_description=README,
-    url='https://www.example.com/',
+    url='https://github.com/Livit/Labster.OAuth2Client',
     author='Labster',
-    author_email='yourname@example.com',
+    author_email='alexander@labster.',
```
Comment (polesye) [inline setup.py]: .com

File: setup.py [hunk 2]
```diff
@@ -9,21 +9,21 @@
 
 setup(
     name='oauth2-client',
-    version='0.1',
+    version='0.1.6',
```
Comment (polesye) [inline setup.py]: 0.1.2

File: setup.py (hunk 1 above)
Comment (auraz) [reply setup.py]: > .com

done

File: setup.py (hunk 2 above)
Comment (auraz) [reply setup.py]: > 0.1.2

done

**Pull request merged**